src/components/Login/RegisterForm.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Form, Icon, Input, Button, Select, message } from 'antd'
+import { register } from '../../services/user'
```

Comment:
The reference path could be referred from 'src' as absolute paths, in this specific case, could be `import { register } from 'services/user'`. No need to refer it with a relative path.

Reply:
```suggestion
import { register } from 'services/user'
```